lib/class-factory.js

```diff
@@ -1386,30 +1386,28 @@ dispatcherPrototype = Object.create(Function.prototype, {
   invoke: {
     value (receiver, args) {
       const overloads = this._o;
+      const methodsThatCanBeInvoked = overloads.filter(method => method.canInvokeWith(args));
+      if (methodsThatCanBeInvoked.length === 0) {
+        throwOverloadError(this.methodName, this.overloads, 'argument types do not match any of:');
```

Comment:
Should we use `this.overloads` here when we have the `overloads` local variable right above us? (If it's the same -- I can't remember, but I think so.)

Reply:
you are right, `this.overloads` and the `overloads ` local variable have the same value.